spec/modules_spec.rb

```diff
@@ -0,0 +1,20 @@
+RSpec.describe 'Wprof Generate Profiling' do
+  context 'When include Wprof to a class' do
+    before do
+      allow(Rails).to receive_messages(wprof: { custom_methods: ['foo_method'] })
+      require 'support/mocks_and_stubs/class_for_modules'
```

Comment:
I think you can use contexts or something for this. It's not a good practice to use plain old require usually

Reply:
That have a good reason. How I need to wrapper a mock class for testing custom methods, I need first set custom methods options and then make a require of class_for_modules. That is it because, these mock class call Wor-Prof for wrapper the method configured (and the configuration of 'custom_methods' must be before it. I didn't see other way for do it, so, if you have another idea for that work on without these 'require' I'm all ears! (Contact me through slack).